internal/provisioner/model/model.go

```diff
@@ -696,6 +692,7 @@ func New(cfg Config) *Contour {
 					},
 				},
 			},
+			KubernetesLogLevel: 1,
```

Comment:
I think the default should stay as it was (i.e. 0)

Reply:
i find the `New` only used in unittests, so set the `KubernetesLogLevel` to `1`, I will change it

Reply:
Ah, I see. OK, let's get this in as-is, and maybe do some follow-up refactoring here. Thanks for the clarification!